specs/consul/SOURCES/consul-client.init

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# consul-client        init file for starting up the consul-client daemon
+#
+# chkconfig:   - 20 80
+# description: Starts and stops the consul-client daemon
+
+###############################################################################
+
+source /etc/init.d/kaosv
+
+###############################################################################
+
+kv[prog_name]="consul-client"
+
+kv.readSysconfig
+
+binary=${BINARY:-/usr/bin/consul}
+conf_file=${CONF_FILE:-/etc/consul/client/config.json}
+
+kv[pid_file]="/var/run/consul-client.pid"
```

Comment:
You use `/var/run` as a directory for the PID file, but run service under `consul` user. `/var/run` can be used only if daemon works under root user, otherwise, you should use `/var/run/consul` directory for PID files.

Reply:
Yeah, that is my bad. Fixed (4d345da0f53facdc92221208165633b6398232cd).